### ISSUE TITLE
umb-toggle-group does not have an accessible name

### ISSUE THREAD

**Issue by lordscarlet** [umbraco/Umbraco-CMS#8716]
The `umb-toggle-group` directive does not have a `<label>` associating the `item.name` with the associated `button` (generated by `umb-toggle`).

## Umbraco version

I am seeing this issue on Umbraco version:  8.6.1

Reproduction
------------

Navigate to `/umbraco/#/users/users/group/-1?create=true`

Under _Default permissions_ investigate the toggles and their associated names.

### Expected result

The toggle should have an associated name. Because the UI presents one, populated by `item.name`, there should be a `<label>` around that name with a `for=` attribute indicated the id of the button that generates the toggle.

### Actual result

There is no association between the two elements

**Comment by umbrabot** [comment]
Hi @lordscarlet,

We're writing to let you know that we would love some help with this issue. We feel that this issue is ideal to flag for a community member to work on it. Once flagged here, folk looking for issues to work on will know to look at yours. Of course, please feel free work on this yourself ;-).  If there are any changes to this status, we'll be sure to let you know. 

For more information about issues and states, have a look at [this blog post](https://umbraco.com/blog/the-umbraco-issue-tracker-process/)

Thanks muchly, from your friendly Umbraco GitHub bot :-)

**Comment by nul800sebastiaan** [comment]
Fixed in https://github.com/umbraco/Umbraco-CMS/pull/8826
